crates/telio-nurse/src/aggregator.rs

```diff
@@ -187,26 +174,32 @@ struct AggregatorData {
     relay_segments: Vec<ConnectionSegmentData>,
 }
 
-// A container to store the connectivity data for our meshnet peers
+/// A container to store the connectivity data for the Meshnet peers used by Nurse
 #[allow(dead_code)]
-pub struct ConnectivityDataAggregator<W: WireGuard> {
+pub struct ConnectivityDataAggregator {
```

Comment:
Not arguing for or against either option (yet), just curious, why remove the generic argument in favor of `dyn`?

Reply:
A very good thing :+1:  no need to propagate the generic argument up to the top structure.